tests/system/src/test/java/test/javafx/stage/FocusedWindowTestBase.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package test.javafx.stage;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.control.TextField;
+import javafx.stage.Stage;
+import javafx.stage.WindowEvent;
+
+import java.lang.ref.WeakReference;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import junit.framework.Assert;
+import org.junit.Ignore;
+import test.util.Util;
+
+@Ignore
+abstract public class FocusedWindowTestBase {
+
+    static CountDownLatch startupLatch;
+    static Stage stage = null;
+
+
+    public static void initFXBase() throws Exception {
+        startupLatch = new CountDownLatch(1);
+        Platform.startup(startupLatch::countDown);
+        Platform.setImplicitExit(false);
+        Assert.assertTrue("Timeout waiting for FX runtime to start",
+                startupLatch.await(15, TimeUnit.MILLISECONDS));
+    }
+
+    static WeakReference<Stage> closedFocusedStageWeak = null;
+    static Stage closedFocusedStage = null;
```

Comment:
These two can be instance fields (which is usually preferred for test variables).

Reply:
They are called from the following static method:
```
    @BeforeClass
    public static void initFX() throws Exception {
        initFXBase();
    }
```

Reply:
No, these two are only used from `testClosedFocusedStageLeakBase`, which is an instance method. I removed `static` while testing it and it compiles and runs fine.

Reply:
You are right, I've probably picked up the other 2 static variables.